newton/_src/solvers/kamino/_src/linalg/factorize/rcm_batch.py

```
@@ -0,0 +1,486 @@
+        b, i = wp.tid()
+        if b >= num_blocks:
+            return
+        n_b = dims[b]
+        if i >= n_b:
+            return
+
+        vb = vio[b]
+        mb = mio[b]
+
+        # Per-vertex init.
+        level[vb + i] = int(-1)
```

Ruff (0.15.10)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)